karton/core/main.py

```diff
@@ -144,10 +144,23 @@ def configuration_wizard(config_filename: str) -> None:
     log.info("Saved the new configuration file in %s", os.path.abspath(config_filename))
 
 
-def print_bind_list(config: Config) -> None:
+def print_bind_list(config: Config, output_format: str) -> None:
     backend = KartonBackend(config=config)
-    for bind in backend.get_binds():
-        print(bind)
+
+    if output_format == "table":
+        # Print a human-readable table-like version
+        print(f"{'karton name':50} {'version':10} {'karton':10}")
+        print("-" * 72)
```

Comment:
Maybe `shutil.get_terminal_size()`? We can still set a max width to 72, but maybe we can set shorter padding on narrower terminal?